### DISCUSSION TITLE
Perfect place to put extension listings?

### DISCUSSION THREAD

**Discussion by Interpause** [camenduru/sd-webui-extension-records#2] [Ideas]
Since this repository is called sd-webui-extension-records, it might make sense to move tracking of extensions here? That way, it will also be easier for extension authors to update their extension listing when they have a major update.

**Comment by ClashSAN** [comment]
I have sent a transfer of ownership to auto. but we have to see if he wants extension index here, or not.

**Reply by ClashSAN** [reply]
he is preferring not to, currently.

**Reply by Gerschel** [reply]
Yeah, it makes sense to keep it containerized with the main project, as it's a main feature of the project. Helps to keep it in sync.

**Comment by WildBanjos** [comment]
So, given that, what is the webui team's preferred method to add extensions to the index and list page? I've seen people add discussions, make posts in existing discussions, create issues, etc.

**Reply by ClashSAN** [reply]
here's some older posts.
https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/1426
https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/3134 (I only saw this months later, unfortunate)

https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/6022
https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/7023

I had just assumed to avoid adding extensions with duplicate functionality.

There are 100+ extensions found on github, some are smaller scripts, and some are now broken. User expectations are quite high, so maybe when making these they could be rebranded as auto-installable custom scripts?
